install/src/Help.hs

```diff
@@ -114,17 +127,31 @@ hieTarget version =
   ("hie-" ++ version, "Builds hie for GHC version " ++ version)
 
 buildTarget :: TargetDescription
-buildTarget = ("build", "Builds hie with all installed GHCs")
+buildTarget = ("build", "Build hie with the lastest available GHC and the data files")
+
+buildLastestTarget :: TargetDescription
+buildLastestTarget = ("build-lastest", "Build hie with the lastest available GHC")
 
 buildDataTarget :: TargetDescription
 buildDataTarget =
   ("build-data", "Get the required data-files for `hie` (Hoogle DB)")
 
 buildAllTarget :: TargetDescription
 buildAllTarget =
-  ("build-all", "Builds hie for all installed GHC versions and the data files")
+  ( "build-all"
+  , "Builds hie for all installed GHC versions and the data files. "
+  ++ buildAllWarning)
+
+buildAllWarning :: String
+buildAllWarning = "WARNING: This command may take a long time and computer resources"
```

Comment:
s/computer resources/a lot of cpu resources/ maybe?

Reply:
No more build-all!